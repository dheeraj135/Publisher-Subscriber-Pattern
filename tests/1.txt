T topic9 oJBLguazHVvCGYc
S topic6 1
T topic2 AnRsGSiPtcjeZtz
U topic7 0
U topic6 1
S topic5 3
T topic4 ZWJxJe6x7KLiPoD
T topic8 3sSmBentPGpN3vg
T topic9 8gPuZXAP0NAMS7V
U topic0 3
S topic9 3
S topic5 2
T topic9 FDVt9y9bVEyAEK3
U topic1 1
T topic7 HF2KCDANidQQN5h
S topic0 1
T topic7 zqYKqB9pQnGVHpn
S topic4 4
U topic8 2
T topic2 Ln78EeHI2PhGyq2
U topic1 4
S topic0 3
S topic1 0
T topic9 pVxS2Ltud4M2tZg
S topic4 0
S topic9 3
S topic1 4
S topic3 0
T topic5 Q4nx61HuEXfhhDv
S topic9 2
T topic4 qsmVS59IHlK34iN
T topic3 zBKp3pXdA26vTKO
T topic4 u7367p5ZQYseDla
S topic3 2
S topic0 0
S topic3 4
S topic7 0
T topic3 diH7UPzyrlRSIv7
U topic4 2
T topic1 ZNRFefuu4zKh6H8
T topic2 OA7xbpX2Fsho7jn
T topic1 DyoxJqiV8ycZSgr
U topic6 0
T topic4 CXBG0Sozz2Wu0sT
T topic0 6NFrqLa3XM3HKaT
S topic8 2
U topic5 0
U topic5 1
T topic9 1upyaUrP8EQ41J4
U topic1 1
T topic6 J5L8Aam6b8m3pAl
U topic7 2
S topic0 0
T topic6 sT5qkFdbznZA38R
T topic6 6dSrd7ERmJL4G0s
U topic7 1
T topic1 pWBcCU5c3shRJ9x
T topic7 Br00ulGeUnJffbF
T topic4 mRbLWFKsMfdMWN3
T topic3 iT0EUFXTKaQCqSW
U topic6 2
S topic3 4
U topic8 0
T topic7 4Be5cLTWjkWqdcY
S topic1 0
T topic5 dCJphpleo9JGjLP
S topic5 1
T topic5 MXfx1lR5GdOlXqF
S topic3 1
U topic5 1
U topic2 4
S topic3 2
T topic6 pY6dEO566R3JlrO
T topic6 9LOhVCVpE2h8qiJ
U topic6 4
U topic6 2
U topic0 0
U topic7 1
U topic8 4
U topic8 2
T topic0 RAe04hR5q7HoLAv
U topic6 0
T topic0 sPbNnd9Qj1n2EWA
T topic1 hpf0A6pC3IAR4fJ
U topic4 1
U topic5 4
U topic2 4
T topic1 0vszrIBQfZGfM29
U topic5 0
U topic4 3
T topic8 PzyHtLz2gmjaaMy
U topic2 0
U topic7 2
T topic9 tDZiCTgjFYfJBma
U topic3 0
U topic2 0
S topic6 2
S topic6 2
T topic5 2lIpr4gzWn9g1vB
S topic8 2
S topic0 1
T topic5 O47PHaCAlrydX44
S topic5 3
S topic9 0
S topic1 4
T topic5 Nx0U59HtXlGLA6O
S topic0 1
T topic4 lC3CKWkBUELeTiQ
T topic4 69nJTP99OdHQYla
U topic1 4
S topic8 3
S topic6 1
U topic5 3
T topic3 tT2gzVpE7l876Eb
U topic4 2
U topic4 0
T topic6 KARDT3XViQTZEpe
S topic3 2
T topic5 KMKE80xsCGDRXXY
S topic4 3
U topic0 3
U topic2 3
T topic0 e5DyucL0quHzBlB
S topic4 0
T topic1 4ETZgn00IDK8lRX
S topic9 4
T topic8 eF8aHJJH6denfQq
U topic4 0
S topic3 0
T topic2 hMSHqQUpSS0cA3n
S topic3 0
U topic6 1
U topic0 4
U topic7 0
S topic8 2
T topic0 bZE8rLOMxHZPa4M
T topic4 H86Xr6cPLlSvn3d
S topic9 1
S topic8 4
S topic4 4
T topic9 4CXB4aA9nr2jVUb
U topic5 0
T topic5 fkouzHS0154ECe0
T topic1 77DVSODSaWn6JgC
U topic5 2
U topic8 1
T topic3 iZtYklcXHgHSAnK
U topic5 3
T topic8 6Azq7JV8Ri2JkFQ
T topic4 bVzRaDbLZVgekjT
S topic9 3
T topic9 o0F6jBvQ9K5xtZG
T topic6 E3aefKMcnxf2lvR
U topic9 4
T topic1 mhsjIBIe4rP3cIL
S topic6 4
S topic6 0
T topic4 hpzszRsBj7GZGlV
S topic2 2
S topic9 2
T topic1 eL2FZexnqbrrOqz
T topic2 EUBVvG6PyY1PeVN
S topic3 3
T topic9 3elbPMGI8oOPWW7
T topic8 bcpK042g9ScMlQB
S topic5 1
T topic2 Punck9vIEU5RcJ3
U topic2 2
T topic4 5RQdk6CvcueLCuc
T topic7 Pfp3xS266ELtb5J
T topic4 sc6hCzcpFn0cYH5
S topic5 0
T topic0 nj2gk57lTI2N7WH
T topic5 McITBc8THsZ3YWZ
U topic3 1
S topic7 0
U topic6 3
T topic8 6J87Z9b1GPfRFET
U topic5 2
T topic4 pLeoyS4xxP4kBiW
U topic3 1
U topic8 2
U topic6 3
T topic9 ukGvIzi5JGuU5WY
T topic9 ulvW9s6JKfzjaNF
U topic3 0
U topic8 3
U topic0 4
T topic6 Bhu0BvknZkmaGbO
T topic3 pC6dy9mCnGApdrA
T topic9 KBKbmJ2pYAKTX1r
U topic1 0
T topic0 5sx0H9XN67jBzCo
T topic9 f30vJet4XiNaEoI
U topic9 3
T topic6 vKUbnXeCY4iyagj
U topic9 4
U topic1 2
T topic3 vV1hlPOmBH2D316
S topic5 0
T topic4 msl2cHdfRRBJg73
U topic3 3
U topic2 3
T topic5 ZNMOEOJdPTAY0ya
U topic2 3
T topic4 eZubslgjUIFiMQA
U topic3 4
U topic4 3
T topic1 vShp6JnbRIRRfM4
U topic1 4
T topic2 l3u1sIcD45LoBYN
U topic4 4
T topic8 ZdNj0iGNMuX94IC
T topic7 sKfIPtJIGje9z4L
U topic4 4
S topic9 3
U topic0 2
U topic0 1
S topic2 1
U topic9 1
U topic0 2
U topic8 1
S topic3 4
U topic8 3
T topic7 sZBUAcAPKvP2KPM
U topic4 1
T topic6 7glOs1z4NXn5qoO
T topic3 VTPHCRz9zrr4rdL
S topic8 0
T topic4 lM276BlUv6zM30U
S topic7 4
S topic5 3
S topic8 0
S topic1 3
U topic5 2
T topic1 miz4XqmvcmnI1Oa
T topic8 QPQ08cFs1ByX2QQ
T topic9 PloQbHbhGPEJqz2
T topic1 Z5tTlmn1dz6b9Cf
S topic9 3
U topic5 0
U topic4 3
S topic6 3
S topic3 1
U topic7 0
T topic9 6BxU0k9IZFhzX8m
T topic9 R9M0dXrNQOBQEBy
T topic5 In3B7zy1D93bqWB
T topic3 7yfRsXCsaFUjLjx
T topic1 vMBdgjMJ71xMFRL
S topic9 3
S topic1 3
U topic8 1
U topic2 3
U topic9 2
U topic9 1
T topic3 z6FV5F1PPGakuIF
S topic1 2
U topic4 3
U topic6 0
S topic9 4
S topic9 4
S topic8 2
U topic5 3
T topic5 xRNgNR1D50F8S98
S topic4 0
S topic5 4
U topic1 1
U topic1 1
S topic4 3
S topic3 1
S topic7 3
T topic2 Aqy3h1SpOi9B70O
U topic6 0
U topic2 3
S topic0 2
U topic7 4
T topic3 IgnO3uFvp9GhqP8
S topic7 4
T topic6 MKKzPipfG0B7HSl
S topic5 4
T topic5 R45T4AtYhOvOZ6y
U topic6 1
U topic7 1
S topic2 0
S topic5 3
S topic5 2
S topic8 2
S topic1 0
U topic2 0
S topic0 4
U topic4 3
T topic6 uZEEcKWmWoY4Krj
S topic1 3
U topic4 2
S topic6 1
S topic9 0
U topic6 0
T topic4 6NnJhKlHUqE8UQl
T topic3 907lYhr2Ne4tIgN
T topic6 CNo4UJkJ5t9k1il
S topic6 1
S topic2 4
T topic3 MjdHRv6S61PfrKA
U topic1 3
S topic7 3
S topic1 4
S topic1 2
U topic0 1
T topic5 za2k5XBa6xXTEzG
U topic1 2
S topic2 1
S topic4 1
T topic6 uUUc6TeGRfverkr
U topic7 3
U topic3 2
U topic9 4
S topic2 0
S topic5 1
U topic0 3
S topic3 3
S topic3 0
S topic3 1
T topic6 WzPapfNZn9EUtqa
U topic0 2
S topic9 1
U topic2 2
S topic9 2
U topic5 2
S topic8 0
T topic4 oLr1RV5SiHUUILE
S topic3 3
U topic1 3
T topic9 rMSILe9UFoqSWgg
T topic7 t3IqP8t6qOgWyeE
U topic0 3
T topic9 6PkmcfWo5UUgJ0M
T topic3 ryPpsjVlXhcT9DH
T topic5 dBn2nmrLf5qAzUQ
T topic1 gUNRlMnbzsdBoKY
T topic6 i73OBhYAhRs8VDx
U topic1 1
U topic0 3
U topic0 1
T topic9 x7iAr86VXNhavvc
S topic7 0
T topic5 6TqTmtodO17R5Uj
U topic9 1
T topic2 8O81N6ZUxWG5LAh
T topic8 ZDFNy8pWQZGJOWu
T topic4 Z1K5HEPO7P2ArDq
U topic6 1
U topic0 0
T topic0 WlFhd3lZG5CpsSL
U topic0 4
S topic6 2
U topic7 0
U topic5 4
T topic0 kkJ65ovErfNqHPh
U topic6 3
T topic7 ctFaxv7AdVD1kjd
U topic2 1
U topic1 1
T topic3 9aW2oqtgWjzCy7g
T topic2 KnOYsAJDR7X3P4f
T topic3 dKfz4OlaXUbY2oZ
T topic6 ptyfKg0CBCYGI4a
U topic7 1
S topic0 4
U topic2 2
T topic3 T3dgQEzDzpkKzq3
T topic2 lHtzVb2HtAXOerg
U topic8 3
U topic2 1
U topic5 2
S topic9 2
U topic0 0
T topic6 4xV34nzdYHj8McR
S topic9 3
T topic6 2HAcTWFZasJyN5Z
T topic4 JtQr7BFdoBdZirr
U topic1 4
U topic8 4
S topic7 4
U topic3 2
T topic4 JXAQ4kCkqZTSmPU
S topic1 1
U topic4 3
U topic5 0
T topic9 9QcZlfRAA1g2icK
T topic9 Yt2LMemMfXS9rFm
U topic2 3
U topic6 2
T topic4 Z85xAdsl3q2HQCo
S topic4 4
U topic3 4
U topic6 0
T topic3 K0x7GoOSnRz2ZFV
T topic1 KKQBd4czXMKcZWH
S topic8 4
U topic9 0
U topic6 2
U topic9 2
T topic3 98egagLxcBHDeCF
S topic3 1
S topic5 4
U topic4 1
T topic8 x26AUf9adUGZBXz
S topic3 0
U topic2 0
S topic1 0
U topic0 4
U topic8 2
U topic4 1
S topic8 4
T topic4 iAE71xe6Z4SUTdR
S topic4 2
T topic4 vuFTWc2K7RPl7UZ
S topic7 1
U topic9 2
U topic3 1
S topic0 0
T topic9 mBI3FM4UV3X6z9l
S topic6 4
T topic7 gbDWXe6LdsBoJ85
T topic5 g8GFcjdi64U2CWk
T topic6 R2SHqzVQNUJQ1oZ
T topic8 SF7tI4PfL5W1hQ4
U topic6 0
T topic9 EuhLAeJIfbpMzXY
S topic7 1
T topic6 f35uFtooSTv987O
S topic1 2
S topic6 2
S topic7 0
U topic4 3
S topic2 1
U topic2 3
T topic0 UFBFkCH7HfRkPgX
T topic9 kqKZQifrfYlRW0N
S topic0 0
T topic5 ehb8r9EAQk7K47H
U topic4 1
U topic8 0
T topic3 vk0qY3OsKNlTFhA
T topic5 EJkhpceuUvN5TI7
T topic4 yaBoPLurpa3os3B
S topic1 1
U topic1 3
S topic6 1
S topic0 0
T topic2 W4RQd6vlPoAkvxP
S topic7 3
S topic7 1
T topic1 dRsPBv2aHkmyUon
S topic8 4
U topic6 2
U topic2 2
T topic5 GvsrPEtVlMyKCmt
S topic0 1
S topic2 4
T topic0 KBMFsRKj8Ar9jbY
S topic1 1
S topic6 1
S topic2 0
S topic8 1
U topic8 1
U topic9 3
U topic5 2
T topic5 1zMN4FPhMgCL4YD
T topic5 7f36lI3q2CRM18U
T topic1 Q2N1aAhmjsiG36B
S topic4 3
T topic2 Ig9euTJt2oPrz6e
U topic0 1
U topic2 2
U topic9 4
U topic3 2
U topic3 0
T topic9 3WrJ2vAI8WhXzJX
T topic4 XfAOgMxV6mdbUyM
T topic3 GMiUi8uM2VpPGap
S topic7 3
S topic2 3
S topic1 0
T topic5 5NNKj0Vt9ETHh0L
U topic3 0
S topic8 3
S topic7 2
T topic7 SYBZ7DD0eirlj1i
S topic3 2
S topic7 1
T topic5 Ya5cHf79VL4fOYZ
S topic9 2
U topic5 0
S topic9 2
U topic3 1
S topic1 1
U topic5 1
U topic2 0
U topic6 0
T topic5 fYzJAB9ibXW9B8f
S topic3 4
T topic6 SNUglbTAfxnBc5e
S topic0 2
U topic0 0
U topic4 4
S topic1 2
T topic6 ix92ZXK0eHp5f4a
T topic8 xX5YpTKxxAPYOIt
U topic4 1
U topic9 2
T topic6 zO7eJkavLCnDtIs
T topic7 TeYO6jz1yAEN2GD
S topic9 2
U topic8 4
S topic4 2
U topic4 2
U topic0 4
S topic3 1
U topic2 0
U topic3 4
S topic4 4
S topic2 1
U topic7 2
S topic6 1
S topic3 1
U topic8 0
T topic9 KP3absTkl26q7GT
U topic9 3
T topic9 vvcPErvPTHZ5ukn
S topic7 2
U topic6 4
S topic3 0
U topic2 0
S topic1 0
S topic8 2
S topic9 3
T topic3 mMXnc9JOY7XVD4Y
S topic4 2
U topic9 3
U topic6 0
S topic7 2
T topic9 bdJ68Q5CX2trNt4
T topic9 4jFJS6s3Yqi7DLr
T topic1 QgsYLazcfc4ik8e
T topic2 kP16CyupOfRL76D
T topic9 OZ57eZ1AtbDFOl2
U topic8 0
S topic4 1
T topic6 QjYI2UCrvfKZWqP
U topic8 3
T topic4 psTgWMpMT0D7LvZ
S topic0 3
T topic8 4zdaxsC8vkVkFtI
S topic9 3
S topic1 0
T topic8 olpMnya63j4aihT
T topic8 f6AoUK5991VQMDc
T topic6 jYnNkqB4hlz19gB
S topic0 0
T topic6 X1qftWlP6ShxTrj
S topic0 4
U topic3 4
S topic7 2
U topic4 1
U topic8 0
T topic0 bbUzxp9UgeYdc0n
T topic0 DJT1Xzr4J9cOBmG
U topic9 4
T topic6 Xd2rrmgKSUuRM6M
S topic1 1
U topic1 2
U topic9 1
T topic9 uKGEV6KDY4dR2zs
T topic8 rV4avgTvz01fHJV
T topic4 GCDd6VRCuksFAex
S topic5 0
T topic1 xdm0sILMnQfPoYY
S topic0 0
T topic3 uuzRomhXBQQTsqJ
U topic4 0
S topic7 2
S topic9 1
S topic2 3
S topic7 0
U topic9 4
T topic4 BjQClAOfsKLBI73
S topic8 0
U topic9 0
U topic7 3
T topic9 JEIQuNT90dxoE7i
U topic7 3
S topic5 2
T topic0 rUvqenfykpPCqpQ
T topic2 gumaadvKMXl0cxC
S topic2 2
S topic1 1
S topic0 3
T topic5 YN3Pghre32nAUKs
T topic8 x7xdQ2pGtV2RVea
T topic0 V8kh4dsQCWgRrH3
S topic3 3
S topic3 4
S topic9 0
S topic2 0
S topic6 4
T topic1 TSf5IqjfIcUHVgh
S topic1 4
T topic5 C8Cm416x759LY3i
S topic3 2
S topic6 2
U topic6 0
S topic0 4
U topic5 3
T topic9 iH4fGZ74zPW5Y3f
S topic9 4
T topic1 EA2maBhFcZGJJ4N
T topic4 MlUjsBEHGZxz5GO
T topic4 lOBzaifMR1FKB6u
S topic9 1
U topic8 3
S topic8 0
T topic3 lhosxKJjYPmsgdH
S topic6 4
S topic0 3
T topic1 8Xg6aHlrcpi9cvX
U topic0 1
U topic0 3
S topic1 4
T topic1 rIfxQMsCvITcVY2
U topic8 0
U topic3 1
T topic7 IKHeZpoCJv5HFHh
U topic0 4
U topic3 4
S topic5 1
T topic6 bthoZ0fVobdpNtI
U topic2 0
T topic7 8DNLWMNV7REQRHW
S topic3 0
T topic3 hJVCaq5EKpEhYjc
T topic6 6UY5M4CaDeYJeXQ
S topic5 0
T topic9 7xuEj3BNdcDoNAX
T topic3 AVoWpuGojadaOvk
U topic6 4
S topic4 2
S topic9 1
U topic3 2
U topic0 1
S topic9 2
S topic4 1
T topic7 xp45MrYrf6kTFvs
S topic3 4
T topic1 6S2A37Z8sJQlbD6
S topic4 1
S topic9 0
U topic0 4
T topic6 WDvY8sNicfqm78K
U topic3 0
U topic6 0
S topic6 0
U topic6 4
T topic5 4o0KKkKDQRGkr9B
S topic9 2
T topic9 VRPXs9J6YG23IZF
T topic1 PIxJxRMN2z1Gd7x
T topic6 dEdF01gjJxhftRZ
T topic6 6fss7yh6ogyvSHs
U topic2 1
S topic0 0
S topic1 3
U topic3 1
U topic0 3
U topic5 0
U topic0 3
S topic5 0
S topic1 3
U topic2 4
U topic3 2
U topic9 1
S topic4 2
S topic9 3
U topic1 0
T topic6 eySEfUzeVae5i0Y
S topic1 0
T topic6 TA1FVkPLLridp6Q
T topic4 atO7TdjneQONL3q
S topic9 1
S topic0 4
T topic1 BcyJ3tgGt64JfYR
U topic3 3
U topic1 1
T topic4 YsVI7tzQz84kXRt
T topic8 SAZ2ujtKMKgsXBU
T topic6 KcUzWIR5Wulqp1J
U topic6 4
T topic8 2SJOgNbKUVnKoq3
U topic0 4
S topic3 4
S topic3 2
U topic2 2
U topic5 4
U topic2 2
T topic8 RUKVjNhfItpG654
U topic3 2
S topic9 2
T topic0 YKJzjuOMycF0cfZ
S topic3 4
T topic8 7nY9KaPk8EfxqBc
S topic2 1
S topic2 2
S topic0 1
S topic8 0
T topic3 3K5X1mRxCWU7GTp
T topic4 Pdd5KtKkhUffVKy
T topic9 HibHz5VHoYUX9Pr
U topic8 4
T topic8 7nMSkvqoV1R0yon
T topic2 WJjIhIpdM5nI4rf
T topic6 tmIXnUtmXhVarlX
U topic8 3
U topic9 1
S topic0 3
S topic2 0
U topic7 0
T topic3 l97PJcrhpJZ7nCF
T topic3 sTl7k7aoYhpzfuq
U topic0 1
U topic4 4
U topic4 3
U topic7 1
U topic2 3
S topic4 4
S topic0 3
U topic7 2
U topic8 2
S topic8 1
T topic2 IzGmHqjZA1fnZpa
T topic1 MQfocCILxLozSsZ
U topic5 1
U topic4 0
T topic3 9yO1OQLQ3ScmRNq
U topic4 3
T topic2 1Y4Zd13RhSD8Zbl
U topic9 4
S topic8 2
T topic8 IPCZD6RsRlfRtu2
S topic3 4
T topic4 YG33Xxef1l5Pq6c
S topic9 3
S topic0 0
T topic6 14HcQWPBGIMLvMB
S topic1 1
U topic9 0
S topic1 0
U topic0 3
S topic9 1
S topic8 0
T topic9 xoesrst53EFuX08
S topic1 2
S topic4 4
U topic9 2
U topic1 4
U topic6 4
S topic2 3
U topic8 0
U topic2 1
T topic9 yro2KQKczrcvLgW
T topic0 jdXqpT45DP8DhsQ
T topic3 EKTbQJMoL4q8XiS
S topic2 3
S topic2 3
U topic0 1
T topic3 iFenkTaC8WP3DOL
S topic4 4
T topic6 9yUTuh263sQyFeZ
U topic9 0
U topic4 0
S topic8 1
U topic5 3
S topic5 1
S topic3 4
S topic0 1
T topic9 lGYuQtatChHEL6x
U topic0 4
U topic4 1
S topic6 3
U topic3 1
U topic0 4
T topic6 nprBh0mu7XkOyLJ
U topic0 1
U topic3 1
T topic4 mMk4yYKzcFilylp
T topic4 tlMfPp7D2kckaLv
S topic4 1
T topic3 BZgG9t9xQWIReRg
S topic2 2
S topic0 3
S topic3 2
U topic1 4
U topic6 2
U topic9 4
U topic6 2
U topic0 4
U topic2 1
T topic4 Bk9GDl3Fz1L63mt
T topic3 05FHcs4gVYvWlJn
U topic2 0
T topic9 OEKddi0FbXy4Ybc
U topic6 4
S topic3 1
U topic0 2
U topic3 3
U topic0 1
T topic5 bpvfrfx5Ujcf8ZV
S topic6 4
S topic3 3
S topic1 1
T topic2 Wax8qHLoPSVMekU
T topic4 G5ipNLJHEONY6mL
S topic0 3
T topic4 UHTxpSojX4uE5xA
S topic4 2
T topic7 nA6H4zhzcq0tNcf
T topic5 GcpLZVSBtYr3Y5l
U topic4 4
T topic9 68xUy8o99uueD9a
U topic0 3
U topic0 1
U topic2 4
U topic4 4
T topic3 WSg80KVLy4WhVLO
S topic6 2
T topic5 mjmfudUQRvb8frC
U topic3 4
U topic2 3
U topic3 4
T topic2 OALZLJYKk1JI0x2
T topic7 qLML9jJOvFR84ZC
S topic8 4
T topic6 y2zGN3ebFvId8lA
T topic6 BtMiRcppAZUsdWQ
S topic6 2
T topic5 WOO7P1OCxW6RtqQ
T topic9 lnkmCISzCbZcSRW
U topic4 4
S topic4 2
S topic0 4
T topic4 UxQ9jxZ0IqAIKop
T topic8 LqYa4RSctimTmrO
S topic4 1
U topic3 4
T topic9 c7Tgtbe3dj4Ih3z
U topic2 4
T topic4 qgWzXhnYUmKDkLD
T topic1 XcRkJmI5grrq6Dn
U topic5 1
T topic7 9k9EeAkrcMVSi2p
S topic7 3
T topic4 BEbAG5yRRloAOsN
T topic4 aJF857BvyM4FLpL
T topic1 59FLRmNAkbrFTL6
T topic3 ge3K3gKBYgO8G3T
U topic0 1
U topic5 3
U topic0 0
U topic5 0
T topic7 TvnjJFxoBKNa09u
T topic7 qGIo684bB249CzE
T topic0 dFQzUTrzal1srfj
U topic6 1
S topic5 2
U topic6 4
S topic3 2
T topic6 L1XugTKTrTed7sF
U topic7 0
T topic6 8DmMcQsbWrbB19V
S topic2 2
T topic5 ULUXOnSbKtRqPU7
T topic2 yW0A338yxcENsdi
U topic8 2
U topic1 1
S topic7 2
S topic9 2
U topic1 2
U topic1 4
U topic9 4
U topic5 4
U topic4 0
U topic3 4
U topic2 4
T topic7 rzq4oAUGY0jleGL
T topic1 j7Y0FdOqjWforiU
T topic8 qnffP7G4h0RWkD9
S topic0 4
T topic9 tDBWXErYUcmdEtY
U topic2 0
S topic9 1
T topic9 840P0x5c68TmdnT
T topic5 6DQmO19EnUAvsNd
S topic6 3
T topic8 Bda79LcQDjKEKpZ
U topic2 1
U topic0 0
U topic9 2
T topic8 C4t5AK3BuXd1n8p
U topic8 3
S topic8 0
T topic3 FHMpDXAz8eYYeFk
S topic4 2
U topic7 0
T topic2 4qhXLy4YP1zZETH
S topic8 3
T topic3 qI91u70EMXU4PZY
S topic3 0
S topic8 0
T topic2 GclrZY6bhJBrQcT
S topic6 3
S topic6 4
S topic9 3
S topic1 3
U topic0 1
S topic0 0
U topic0 2
U topic4 3
U topic0 1
U topic3 0
S topic8 3
S topic0 3
U topic5 4
T topic6 lJA5TEIv4aCeHe8
T topic8 Ix6Rk9aiHZ7fOJC
U topic1 1
T topic6 Zvzya9V4R4if5Jg
T topic3 mJng0PBGiMYNYPV
S topic5 3
U topic5 4
T topic6 vOIppsOigAnnpEs
U topic9 1
T topic2 BNtQRn55LjeSauv
U topic3 4
S topic7 0
T topic4 tOFf44SDe1xgjdk
U topic0 2
S topic3 2
U topic1 2
T topic4 puPKOgsWXIcMUVY
U topic6 2
S topic1 4
U topic6 2
S topic4 1
U topic4 3
S topic9 2
T topic4 uHUnU4GdZXycVmf
S topic3 4
T topic1 WbN4vgSsblAFhLb
T topic5 Lzu9BuEvWsKaCVp
T topic2 Go3JVXiEO0451Wx
T topic1 S0nJsGl3dN5qtcV
S topic2 3
U topic1 0
U topic5 3
S topic5 0
U topic0 3
U topic0 3
U topic4 1
S topic7 4
T topic2 lbm45TaTb2YnUuu
S topic1 2
S topic7 4
S topic3 0
U topic2 1
U topic2 1
S topic6 0
S topic4 3
U topic8 1
U topic8 2
T topic3 0hAN3sN3VpalA3q
S topic8 2
U topic9 1
S topic0 3
S topic0 2
T topic0 NyWl3jDJuRvzMU6
T topic5 4uluD9ADU3XYXhE
U topic2 3
T topic5 2qmnSU0MMp9JfuH
T topic1 VTOomKyFLUTSc5P
S topic2 4
T topic6 68H29BMUFWEGzXq
U topic8 0
S topic3 0
T topic5 PKlGU8vFy4i3uvT
U topic2 0
U topic8 4
S topic3 0
T topic5 dVFDZNxuCXtpfoy
U topic4 2
S topic2 2
