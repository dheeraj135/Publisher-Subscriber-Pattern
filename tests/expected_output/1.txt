TvE77YfY4brMbUk
jnPGHBxzuk487IO
9NOVFuVpjeieu7f
cRPrxCXeAWngK9n
AajJ6xd8d4WmsvC
EjUaxWo0SCev67l
Gsdbsvj0z2X8P5b
ilTzJc0fjys3WVv
nFbBYzLYcryHMzW
zFiR95FMsxnuasK
6e7C117WX7832bX
2jMlJFEhZIR0JDt
IsCSlBKvLYvBtej
eaHHyP22FPJ1UOm
tvAFM5p12XfyXzp
ena2xGm79CIHiq6
d88ylpzPOoqtJI7
h4XLmb5ET7ohDPz
AGIRVByMoV2etdd
XnL0XY1zQq4UPhb
0avUGqdPjhp3tuY
